src/EventStore.Client/ChannelCache.cs

```diff
@@ -1,17 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
-using System.Threading.Tasks;
-
+﻿using System.Net;
 using TChannel = Grpc.Net.Client.GrpcChannel;
 
 namespace EventStore.Client {
 	// Maintains Channels keyed by DnsEndPoint so the channels can be reused.
 	// Deals with the disposal difference between grpc.net and grpc.core
 	// Thread safe.
 	internal class ChannelCache :
-		IDisposable, // for grpc.net we can dispose synchronously, but not for grpc.core
+		//IDisposable, // for grpc.net we can dispose synchronously, but not for grpc.core
```

Comment:
accidental comment?

Reply:
correct, since it now implements IAsyncDisposable